universum/modules/vcs/github_vcs.py

```diff
@@ -84,7 +89,9 @@ def define_arguments(argument_parser):
 
         parser.add_argument("--github-installation-id", "-gti", dest="installation_id",
                             metavar="GITHUB_INSTALLATION_ID",
-                            help="Calculated out of webhook payload (real help coming soon)")
+                            help="GitHub installation ID identifies GitHub App user. Can be retrieved from "
```

Comment:
In my opinion "GitHub App user" is not correct term.
Installation ID identifies specific app installation into user account or organization.